mod/account/set/all/password_input.haml

```diff
@@ -0,0 +1,3 @@
+%input.form-control.d0-card-content._pw-input{name: "card[subcards][+*password][content]", type: "password", placeholder: "Password"}
```

Comment:
We don't want the cardname ("*password") hardcoded here. That would mean that if someone renamed the password card, the signin would break.  Can we use ":password" instead? The colon indicates a codename, which will not change even if the cardname does.

Reply:
Did you see my above comment? It breaks when I change it. See above with screenshot.